bbc_clean/treerez/treerez.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+"""
+TreeRez
+convert and resize
+"""
+
+__author__ = "Frenchfaso"
+__version__ = "0.1.0"
+__license__ = "MIT"
+
+
+import logging
+logging.basicConfig(level=logging.INFO)
+from PIL import Image
+import numpy as np
+import os
+from random import sample
+import argparse
+
+
+def get_patients_sample() -> list:
+	""" Get patients folders from the source folder and return a list of random samples."""
+	
+	patients = list([obj.name for obj in os.scandir(source_folder) if obj.is_dir()])
+	patients_sample = sample(patients, quantity)
+	logging.info(f"Sampled {len(patients_sample)} patients out of {len(patients)} total.")
+	return patients_sample
+
+
+def get_scans(patients_sample) -> list:
+	""" Return a list with all the scans of the sampled patients """
+	
+	scans = []
+	for patient in patients_sample:
+		xray_types = os.scandir(os.path.join(source_folder, patient))
+		for xray in xray_types:
+			if(xray.is_dir()):
+				for img in os.scandir(xray.path):
+					if(img.name.endswith(('tif','tiff'))):
```

Comment:
What about upper case extensions? Will this catch *.TIF files as well? If not just make it case insensitive.